docs/python/testing.md

```diff
@@ -186,7 +186,7 @@ You can run tests using any of the following actions:
 
   - You can also run a selection of tests through the Test Explorer. To do that, `kbstyle(Ctrl+Click)` (or `kbstyle(Cmd+Click)` on macOS) on the tests you wish to run, right-click on one of them and then select **Run Test**.
 
-After a test run, VS Code displays results directly in the editor as gutter decorations. Failed tests will also be highlighted in the editor, with a Peek View that displays the test run error message and a history of all of the tests' runs. You can press `kbstyle(Escape)` to dismiss the view, and you can disable it by opening the User settings (**Preferences: Open Settings (UI)** command in the **Command Palette**) and changing the value of the `Testing: Automatically Open Peek View` setting to `never`.
+After a test run, VS Code displays results directly in the editor as gutter decorations. Failed tests will also be highlighted in the editor, with a Peek View that displays the test run error message and a history of all of the tests' runs. You can press `kbstyle(Escape)` to dismiss the view, and you can disable it by opening the User settings (**Preferences: Open Settings (UI)** command in the **Command Palette**) and changing the value of the `testing.automaticallyOpenPeekView` setting to `never`.
```

Comment:
Hi @lemonez The setting is being referenced by its Settings editor text. I think we should leave it as is but since it is a UI string, our convention is to use **bold** rather than `inline code block` formatting.

![image](https://user-images.githubusercontent.com/12818376/152417169-6d2f88b7-d28e-41ea-855a-eaea028c4eba.png)

Reply:
Hi @gregvanl , that makes a lot of sense! The formatting is what threw me, and also the fact that some of the other settings are referenced by the JSON key string so it was unclear which was being referenced. So yeah I'd agree, I think **bold** formatting would do the trick; I'll make that change. Thanks for the quick response.